packages/components/src/drop-zone/index.tsx

```diff
@@ -157,7 +157,10 @@ export function DropZoneComponent( {
 				exit={ disableMotion ? 'show' : 'exit' }
 				className="components-drop-zone__content"
 			>
-				<motion.div variants={ foreground }>
+				<motion.div
+					variants={ foreground }
+					className="components-drop-zone__content-foreground"
```

Comment:
Is this change and the associated css changes something that's still needed? I couldn't see that the code in this PR uses this component.

Reply:
Oops, yeah, it's something I forgot to delete.